[IMP] ecommerce: Update of the page to reflect all changes in 15.5 (#2429)
odoo/documentation: content/applications/websites/ecommerce/getting_started/catalog.rst

===========================
Customize your catalog page
===========================

When selling online, your product catalog is a crucial part of your efforts. It helps
showcase all products being currently available while making it easier for everyone to
find the product they need.

The more information you give about your products, the better. Photos, descriptions, prices, so many
aspects that can help increase sales. With Odoo, all of that is managed from one single view.

.. _ecommerce/started/catalog/catalog:

Product Catalog
===============

When opening your **shop** for the first time, all items are published one after the other. You need

> to customize it so that you display the information you need.

Reviewer: ```suggestion
to customize it so that the information you need is displayed.
```
It might just be me, but I find the phrasing a bit weird... Here's my suggestion.

.. image:: catalog/shop-overview.png
   :align: center

>    :alt: Overview of the catalog page without any customization nor navigation options.

Reviewer: ```suggestion
   :alt: Overview of the catalog page without any customization or navigation options.
```
"Without" is not a negative word, therefore "nor" cannot be used here :)

> You can display more information or change the page layout with the :guilabel:`Customize`

Reviewer: ```suggestion
You can display more information or change the page layout through the :guilabel:`Customize`
```
or "in","from", etc..

menu. To access it, go to :menuselection:`Edit --> Customize`.

> Once there, many customization options appears:

Reviewer: ```suggestion
Once there, multiple customization options appears:
```
or "a variety of"

- :guilabel:`Layout`: Allows to choose how products are displayed. It includes the number of
  products on the   page and per line, the style, and the size of your product pictures.
- :guilabel:`Prod. Desc.`: Allows to display the products' description if there's one.
- :guilabel:`Left Panel`: Allows to add more navigation options, such as accessing specific product
  categories or attributes (colors, sizes, etc.).
- :guilabel:`Top Bar`: Allows to add more display options for customers, such as how product are
  sorted by, or the layout of the page (grid, list, etc.).
- :guilabel:`Default Sort`: Allows to choose how products are sorted when opening the shop.
- :guilabel:`Buttons`: Allows to add quick actions, such as add to cart, add to favorite, etc.

> Let's see some of those in action.

Reviewer: ```suggestion
Let us see some of those in action.
```
I see more after this one, I'm not going to mark them, but keep it in mind :)

.. _ecommerce/started/catalog/layout:

Page layout
===========

Under the :guilabel:`Layout` section, you can change several display options. For this example,
let's change the number of products per page from 20 to 21, and the number of products per line
from 4 to 3 with the :guilabel:`Size` option.

> For the :guilabel:`Style` and :guilabel:`Images Size`, let's choose :guilabel:`Cards` and

Reviewer: ```suggestion
For the :guilabel:`Style` and :guilabel:`Images Size`, let us choose :guilabel:`Cards` and
```

:guilabel:`Landscape (4/3)` to add borders to our pictures and reduce the size of each block.

.. _ecommerce/started/catalog/panel:

Left Panel
==========

To give your customers more navigation options, you can use the :guilabel:`Left Panel` options.
Let's display :guilabel:`Categories` and enable :guilabel:`Collapse Categories` to add a navigation
menu for customers.

> Since giving the more information about your products is important, let's also enable the

Reviewer: ```suggestion
Since giving more information about your products is important, let's also enable the
```
I think you left a "the" when rewriting the sentence

:guilabel:`Prod. Desc.` option to display the description of each product. The option can be found
just above :guilabel:`Left Panel`.

.. _ecommerce/started/catalog/cart:

Quick add to cart
=================

With the :guilabel:`Buttons` options, let's add several possibilities. Click on
:guilabel:`Add to Cart` and :guilabel:`Wishlist` to optimize the customer journey.

.. _ecommerce/started/catalog/highlight:

Highlight a product
===================

> While having the website builder opened, click on a specific product to customize its block.

Reviewer: ```suggestion
While having the website builder open, click on a specific product to customize its block.
```
